packages/upload-client/test/store.test.js

```diff
@@ -62,16 +62,21 @@ describe('Store.add', () => {
       channel: server,
     })
 
+    let progressStatusCalls = 0
     const carCID = await Store.add(
       { issuer: agent, with: space.did(), proofs, audience: serviceSigner },
       car,
       {
         connection,
+        onUploadProgress: (status) => {
+          progressStatusCalls++
```

Comment:
nit: will you assert that `typeof status.loaded === 'number' && status.loaded > 0`. That seems like the most useful property for progress bar, but could to assert presence/type of any other properties we expect to be part of long-term public api for this cb.

Reply:
done!